user/apt-packages.md

```diff
@@ -6,15 +6,16 @@ permalink: /user/apt-packages/
 <div id="toc">
 </div>
 
-When using the [container based infrastructure](/user/workers/container-based-infrastructure/), `sudo` is disabled
-in user-defined build phases such as `before_install`. This prohibits installation of APT packages via `apt-get`.
-In order to have Travis CI do this on your behalf before `sudo` privilege is removed, use the `addons.apt_packages`
```

Comment:
I tend to delete things like 'In order to'. And in fact you could say something like

    -To install packages before your custom build steps, use the `addons.apt_packages`

instead of that last bullet.